src/python/crash_analysis/stack_parsing/stack_analyzer.py

```diff
@@ -1320,14 +1320,15 @@ def get_crash_data(crash_data, symbolize_flag=True):
       continue
 
     # Sanitizer regular crash (includes ills, abrt, etc).
-    update_state_on_match(
-        SAN_ADDR_REGEX,
-        line,
-        state,
-        type_from_group=2,
-        address_from_group=4,
-        reset=True,
-        type_filter=fix_sanitizer_crash_type)
+    if not is_golang:
```

Comment:
I don't think we should disable sanitizer stacks completely. what if we have a go crash without go stack and only asan stack (we want to create a asan stack signature). i think your intent is to not update with sanitizer stack if a go stack is found. this is easy to do with initing a bool found_golang_crash=false and then set it when you detect golang crash type.

```
if update_state_on_match(
     golang_crash_regex, line, state, new_type=golang_crash_type):
         found_golang_crash = True
          state.crash_state = ''
```

and then doing if not found_golang_crash here.

Reply:
Yes, that is much better, thanks! There is one more example now: https://oss-fuzz.com/testcase-detail/5664480284901376